cetmix_tower_git/readme/DESCRIPTION.md

```
@@ -0,0 +1,4 @@
+This module implements Git Management tools for Cetmix Tower.
+
+For detailed information please refer to the official documentation: https://cetmix.com/tower
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)